fd/example_test.go

```diff
@@ -44,3 +45,24 @@ func ExampleDerivative() {
 	// f'(0) ≈ 0.9999998333333416
 	// f''(0) ≈ -2.999999981767587
 }
+
+func ExampleJacobian() {
+	f := func(y, x []float64) {
+		y[0] = x[0] + 1
+		y[1] = 5 * x[2]
+		y[2] = 4*x[1]*x[1] - 2*x[2]
+		y[3] = x[2] * math.Sin(x[0])
+	}
+	x := []float64{1, 2, 3}
+	jac := fd.Jacobian(nil, f, 4, x, &fd.JacobianSettings{
+		Formula:    fd.Central,
+		Concurrent: true,
+	})
+	fmt.Printf("J ≈ %v\n", mat64.Formatted(jac, mat64.Prefix("    ")))
+
+	// Output:
+	// J ≈ ⎡ 0.9999999999917482                    0                    0⎤
```

Comment:
Maybe make it %0.4v or %0.6v? This level of precision might not be robust to different go editions.